src/public-css/navbar.css

```diff
@@ -53,4 +67,8 @@
 
```

Comment:
You might wanna check what is happening here.

Reply:
Yeah sure! Thanks for pointing it out.